fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreTestBase.java

```diff
@@ -220,8 +229,15 @@ public void commit(FDBRecordContext context) {
             if (logger.isInfoEnabled()) {
                 KeyValueLogMessage msg = KeyValueLogMessage.build("committing transaction");
                 msg.addKeysAndValues(timer.getKeysAndValues());
+                msg.addKeyAndValue(LogMessageKeys.TRANSACTION_ID, context.getTransactionId());
                 logger.info(msg.toString());
             }
+        } catch (FDBExceptions.FDBStoreTransactionConflictException conflictException) {
+            List<Range> conflictRanges = context.getNotCommittedConflictingKeys();
+            if (conflictRanges != null && !conflictRanges.isEmpty()) {
+                conflictException.addLogInfo("conflict_ranges", conflictRanges);
+            }
+            throw conflictException;
```

Comment:
This is somewhat unrelated, but the idea is to hopefully use the `conflictingKeys` feature to help find problems if there are conflicts caused by running tests in parallel. This will ensure the `conflict_ranges` get put into the exception, which will then be logged (see the `ExceptionLoggingDetailsExtension`)